docs/plugins/github-apps.md

```diff
@@ -0,0 +1,78 @@
+# Using GithubApps for backend authentication
+
+Backstage can be configured to use GitHub Apps for backend authentication. This
+come with advantages such as higher rate limits and that Backstage can act as an
+application instead of a user or bot account.
+
+It also provides a much clearer and better authorization model as a opposed to
+the OAuth apps and their respective scopes.
+
+## Caveats
+
+- It's not possible to have multiple backstage GitHub Apps installed in the same
+  GitHub organization be managed by Backstage. We currently don't check through
+  all the registered GitHub Apps to see which ones are installed for a
+  particular repository. We just respect global Organization installs right now.
+- App permissions is not managed by Backstage. They're created with some simple
+  default permissions which you are free to change as you need, but you will
+  need to update them in the GitHub web console, not in Backstage right now. The
+  permissions that are defaulted are `metadata:read` and `contents:read`.
+- The created GitHub App is private by default, this is most likely what you
+  want for github.com but it's recommended to make your application public for
+  GitHub Enterprise in order to share application across your GHE organizations.
+
+A GitHub app created with `backstage-cli create-github-app` will have read
+access by default. You have to manually update the GitHub App settings in GitHub
+to grant the app more permissions if needed.
+
+### Using the CLI (public GitHub only)
```

Comment:
i think this and the ones below should be level 2 `##` right? otherwise they are under `Caveats`